### PR TITLE
README.md: remove circleci badge

### PR DESCRIPTION
**Pull request by andreimatei**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/10513)

### CODE REVIEW

**Comment by tamird** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ![logo](docs/media/cockroach_db.png?raw=true "Cockroach Labs logo")
 
 
-[![Circle CI](https://circleci.com/gh/cockroachdb/cockroach.svg?style=svg)](https://circleci.com/gh/cockroachdb/cockroach) [![GoDoc](https://godoc.org/github.com/cockroachdb/cockroach?status.svg)](https://godoc.org/github.com/cockroachdb/cockroach) ![Project Status](https://img.shields.io/badge/status-beta-yellow.svg) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/cockroachdb/cockroach?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![GoDoc](https://godoc.org/github.com/cockroachdb/cockroach?status.svg)](https://godoc.org/github.com/cockroachdb/cockroach) ![Project Status](https://img.shields.io/badge/status-beta-yellow.svg) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/cockroachdb/cockroach?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (tamird) [inline README.md]: while you're here, can you add some newlinews? I don't believe they affect how this is rendered.

**Review by tamird — APPROVED** [review]

**Comment by jordanlewis** [comment]
Wanna add a TeamCity badge? https://confluence.jetbrains.com/display/TCD10/REST+API#RESTAPI-BuildStatusIcon

**Comment by jordanlewis** [comment]
You can use this link: https://teamcity.cockroachdb.com/guestAuth/app/rest/builds/buildType:(id:Cockroach_UnitTests)/statusIcon.svg

It looks like this: status/statusIcon.svg)

**Pull request closed without merging**